docs/getting-started/app-custom-theme.md

```diff
@@ -228,6 +228,88 @@ const myTheme = createUnifiedTheme({
 });
 ```
 
+## Custom Fonts
+To add custom fonts, you first need to declare the font style following the `@font-face` syntax from [Mui Typography](https://mui.com/material-ui/customization/typography/). After that you can then utilize the `styleOverrides` of `MuiCssBaseline` under components to set the default font. 
+```ts title="packages/app/src/theme/myTheme.ts"
+import MyCustomFont from '../assets/fonts/My-Custom-Font.woff2';
```

Comment:
Really, you could place them anywhere as long as you can reference them. That was just the convention I landed on. I was trying to figure out if there was a webpack requirement for the loaders to work, but couldn't find the reference.

Reply:
Do we have a code structure doc we should reference or also establish this as the convention?

Reply:
I see how they did it here. I'll follow suit.

https://backstage.io/docs/getting-started/app-custom-theme/#create-react-component

Reply:
Yes, that's true but for other's it may not be that obvious. I've often helped people on Discord who didn't realize they could just create a folder to put their theme into. We added a note to make it more clear. One of the things we always need to keep in mind is that many who use Backstage may not be experts in its tech stack. That was and is true for myself.

Reply:
> I see how they did it here. I'll follow suit.
> 
> https://backstage.io/docs/getting-started/app-custom-theme/#create-react-component

Perfect!